pages/api/wallets/[viewWalletPubkey].ts

```diff
@@ -1,40 +1,100 @@
 // Next.js API route support: https://nextjs.org/docs/api-routes/introduction
 import type { NextApiRequest, NextApiResponse } from 'next'
 import { Membership, PrismaClient } from '@prisma/client';
-import { MembershipModel } from '@glasseaters/hydra-sdk';
+import { Connection, clusterApiUrl, Cluster, PublicKey } from '@solana/web3.js'
 
-const prisma=new PrismaClient();
+import { useAnchorWallet, useConnection } from '@solana/wallet-adapter-react'
+import { MembershipModel, FanoutClient, Fanout } from '@glasseaters/hydra-sdk';
 
+const prisma = new PrismaClient();
+
+function validateWallet(cluster, viewWalletPubkey) {
+    const connection = new Connection(clusterApiUrl(cluster as Cluster), 'confirmed')
+    try {
+        let publickey = new PublicKey(viewWalletPubkey);
+        let a = Fanout.fromAccountAddress(connection, publickey);
```

Comment:
Since we already get a `Fanout` object here, I think that a later extension to this as you previously said is to compare the values of the fields in the database with the values of the `Fanout` object for further validation. I'll add an issue for that but let's put it off for now.

Reply:
Yeah, that sounds good